### ISSUE TITLE
How to show images in the SLY's REPL?

### ISSUE THREAD

**Issue by svetlyak40wt** [joaotavora/sly#629]
For SLIME there is a slime-media contrib with [slime-media-insert-image](https://github.com/slime/slime/blob/864ed29afc2e03688d59a23f01979b0a781f1a8b/contrib/slime-media.el#L35) which can be used like this: https://github.com/pszer/emacs-slime-image

But for SLY I can't find anything like that. Am I'm missing something?

**Comment by svetlyak40wt** [comment]
@joaotavora what do you think?

**Comment by joaotavora** [comment]
I think it's a fine idea, but SLIME's contrib isn't immediately compatible, so you need to make something that plugs into `sly-mrepl.el` and `sly-mrepl.lisp`.  I'd make a new contrib.  You can make an in-tree contrib (under `contribs/`) or an out-of-tree one (using the `sly-hello-world` template).

Make this one as an in-source one, i.e. as a PR.  That's because you will probably need to change bits of `sly-mrepl.el` and `sly-mrepl.lisp`.  Good luck, I can't help you much.  Maybe @aadcg can accompany you in this work.

**Comment by aadcg** [comment]
@joaotavora I think I could contribute some time to maintain SLY, since it seems clear that it's hard for you to be on top of everything.

Regarding this issue in particular, I'd be happy to review it, but I'm not keen on working on it.

Greetings @svetlyak40wt, hope you're doing well.

**Comment by joaotavora** [comment]
@aadcg yes.  I'll send you an invitation.  Be careful, last I heard project is used in production by many good friends :-)  Shoot me a mail if in doubt about anything.

**Comment by joaotavora** [comment]
André, feel free to list yourself as current maintainer (or Co-Maintainer maybe) in the relevant source files and in appropriate forums.

**Comment by aadcg** [comment]
@joaotavora thanks! I'll add myself as a co-maintainer as soon as I have done some meaningful work. For the time being, I can't contribute much time but hopefully some.

I'll always ping you when in doubt, and maintain a rather conservative posture.

**Comment by jcguu95** [comment]
@svetlyak40wt Good news! I just implemented what you want :) Thanks for bringing up this request: https://github.com/joaotavora/sly/pull/638
